code_share/code_share/views.py

```diff
@@ -0,0 +1,17 @@
+from django.shortcuts import render
+
+
+def page_not_found_view(request):
```

Comment:
This `view` and the ones that follows this will interpreted as `200` because it's basically returning  a success response instead of raising 404. You need to make the changes as described [here](https://stackoverflow.com/a/15006243/7055424)
I found this while writing tests for the error cases, that's why it's recommended to write test cases for every changes. @theparadoxer02 :)